models/User.php

```diff
@@ -1,7 +1,104 @@
 <?php
 
+require 'Course.php';
+
 class User
 {
+    /**
+     * @return mixed
+     */
+    public function getUFirst()
```

Comment:
Please move the getters and sestters below the constructor for better readability